### ISSUE TITLE
Add Permission info of extensions

### ISSUE THREAD

**Issue by pavi2410** [domhnallohanlon/thunkable_extensions#11]
It will be very good for people (_those who are concerned about the privacy of their users_) to know what permissions do the extensions use, so that they can take better decisions.

**Comment by conorshipp** [comment]
Yes, it's a very good idea. I am going to be doing this, adding a list of permissions the extensions use to the overview tables at the top of each post.
